locale/en-US.properties

```diff
@@ -0,0 +1,11 @@
+tooltip_play= Play
+tooltip_pause= Pause
+tooltip_mute= Mute
+tooltip_unmute= Unmute
+tooltip_send_to_tab= Send to tab
+tooltip_minimize= Minimize
+tooltip_maximize= Maximise
+tooltip_close= Close
+error_msg= Something's gone wrong with this video, Try again later.
+error_link= Open in new tab
+loading_msg= Loading video from
```

Comment:
I'm not sure this will work. This assumes that the domain would come after the phrase 'loading video from', but in right-to-left languages, this won't be the case.

In cases like this, you usually want a placeholder, like `${domain}`, so that localizers can figure out what the complete phrase means, then move the variable around accordingly. You can also insert a comment just above this particular line to explain to localizers what the variable names mean. To work with the localization tools, comments should start with `# LOCALIZATION NOTE:`, like [this](https://github.com/mozilla/testpilot/blob/ab37969de5f7eb52a222f37fee283cf6e773da97/locales/en-US/app.ftl#L59) example from test pilot.

Since we have no locales yet, I'm good to land this for now and file a followup bug to localize it correctly.